test/vector-metadata-constexpr.ll

```diff
@@ -1,15 +1,25 @@
 ; RUN: llvm-as %s -o %t.bc
-; RUN: llvm-spirv -s %t.bc -o - | llvm-dis -o - | FileCheck %s
-; RUN: llvm-spirv %t.bc
+; RUN: llvm-spirv -s --spirv-ext=+SPV_INTEL_arbitrary_precision_integers,+SPV_INTEL_vector_compute %t.bc -o - | llvm-dis -o - | FileCheck %s
+; RUN: llvm-spirv --spirv-ext=+SPV_INTEL_arbitrary_precision_integers,+SPV_INTEL_vector_compute %t.bc
```

Comment:
Why did you add the extensions?

Reply:
I've added test case with the vector of size 1 which is supported only with the `SPV_INTEL_vector_compute` extension, and the second extension is not needed at all.
I'll change the test a bit to avoid using extensions.

Reply:
Thanks!

Reply:
done in 1bc8f8b